patches/content-browser-renderer_host-navigation_controller_impl.cc.patch

```diff
@@ -23,3 +23,11 @@ index 7bbb66fee4f8e14a5f27825406ebeee9ca4bf8e6..8689250625fdb800ab5fe1d5cb88cfe4
      return nullptr;
    }
  
+@@ -3848,6 +3849,7 @@ NavigationControllerImpl::CreateNavigationRequestFromLoadParams(
+     CHECK(!frame_entry || url_to_load == frame_entry->url());
+   }
+ 
++  BRAVE_MAYBE_FILTER_TWITTER_PARAM
```

Comment:
pls name these defines as `BRAVE_<CLASS_NAME>_<FUNCTION_NAME>`

Reply:
I renamed them `BRAVE_<FUNCTION_NAME>` since there's no class in this case.